src/libs/ajax/ajax-common.ts

```diff
@@ -242,4 +242,6 @@ export const fetchWDS = (wdsProxyUrlRoot) =>
   _.flow(withUrlPrefix(`${wdsProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
 
 export const fetchCbas = (cbasProxyUrlRoot) =>
-  _.flow(withUrlPrefix(`${cbasProxyUrlRoot}/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
+  _.flow(withUrlPrefix(`${cbasProxyUrlRoot}/api/batch/v1/`), withRetryAfterReloadingExpiredAuthToken)(fetchOk);
+
+export const fetchCromwell = withUrlPrefix(`${getConfig().cromwellUrlRoot}/api/workflows/v1/`, fetchOk);
```

Comment:
Same comment regarding the pattern here. Also I think we might need to pass the Cromwell proxy url root here to since we would be calling Cromwell inside the app and not the general GCP Terra Cromwell.